### PR TITLE
Document sudo/user change + tags for image

### PR DESCRIPTION
**Pull request by decentral1se**
#### PR Type
- Docs Pull Request

Something for https://github.com/ansible/molecule/issues/1813. As a result of changes in https://github.com/ansible/molecule/pull/1797. Also part of what I agreed to do in https://meetbot.fedoraproject.org/ansible-molecule/2019-03-13/ansible_molecule_working_group.2019-03-13-19.00.html :+1:

### CODE REVIEW

**Comment by decentral1se** [comment]
@gundalow please merge when #1818 is resolved :+1:

File: CHANGELOG.rst [hunk 1]
```diff
@@ -17,6 +17,12 @@ Important Changes
 Other
 -----
 
+* Molecule docker images will use the following convention on tags going forwards:
+  * ``latest``: corresponds to the master branch
```
Comment (gundalow) [inline CHANGELOG.rst]: Do we want to add
```suggestion
  * ``latest``: corresponds to the master branch, which should be viewed as unstable
```

File: doc/source/examples.rst [hunk 2]
```diff
@@ -24,7 +26,7 @@ follows.
         -v /var/run/docker.sock:/var/run/docker.sock \
         -w /tmp/$(basename "${PWD}") \
         quay.io/ansible/molecule:latest \
```
Comment (gundalow) [inline doc/source/examples.rst]:
```suggestion
        quay.io/ansible/molecule:2.20 \
```

**Comment by decentral1se** [comment]
Builds fine:

```
(.venv) ➜  doc (docsfix/dockerfile-v220) ✔ make html
sphinx-build -b html -d build/doctrees  -n -W source build/html
Running Sphinx v1.8.4
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 2 source files that are out of date
updating environment: [] 0 added, 3 changed, 0 removed
reading sources... [100%] installation
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] installation
generating indices... genindex
writing additional pages... search
copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded.

The HTML pages are in build/html.

Build finished. The HTML pages are in build/html.
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -17,6 +17,12 @@ Important Changes
 Other
 -----
 
+* Molecule docker images will use the following convention on tags going forwards:
+  * ``latest``: corresponds to the master branch, which should be viewed as unstable
```
Comment (ssbarnea) [inline CHANGELOG.rst]: seems wrong: latest should point to last release (not last pre-release)
master should point to last build from master, what counts as unstable

File: doc/source/installation.rst [hunk 4]
```diff
@@ -69,6 +69,22 @@ some benefits:
 * allows molecule installation at user level without even needing to have
   the script in ``PATH``.
 
+Docker
+======
+
+We publish molecule images via `quay.io`_ where the following tags are available:
+
+  * ``latest``: latest master branch build, which should be viewed as unstable
```
Comment (ssbarnea) [inline doc/source/installation.rst]: same comment as above

Comment (gundalow) [reply doc/source/installation.rst]: Not currently possible, hence suggesting people use named versions. We will look at fixing this in 2.21

File: CHANGELOG.rst (hunk 3 above)
Comment (gundalow) [reply CHANGELOG.rst]: Not currently possible, hence suggesting people use named versions. We will look at fixing this in 2.21

**Pull request merged**